### DISCUSSION TITLE
Retire as a Utilities Maintainer

### DISCUSSION THREAD

**Discussion by sophiewigmore** [paketo-buildpacks/feedback#173] [Utilities Team]
In an effort to focus on other Paketo maintainerships I have, I would like to retire as a maintainer from the Utilities team. I believe the Utilities Maintainer team will remain  in good shape in my absence since there are a few other maintainers on this team already!

**Comment by ryanmoran** [comment]
Thank you for your contributions! I've included you in the list of emeritus maintainers, and reduced your current status to Contributor.

**Comment by sophiewigmore** [comment]
Thank you!
